packages/node-modules-inspector/src/app/pages/grid.vue

```diff
@@ -2,6 +2,7 @@
 import type { PackageNode } from 'node-modules-tools'
 import { computed } from 'vue'
 import { payloads } from '~/state/payload'
+import { rawPublishDates } from '~/state/publishDates'
```

Comment:
not sure if we want the raw state here

Reply:
I think instead of passing the object to the props, we could just expose a global lookup function.

This way we could also do on-demand fetch.